R/utils.R

```diff
@@ -45,7 +45,15 @@ register_functions_to_cluster <- function(cluster) {
         'span',
         'compute_uniq_grp',
         'predict_smoothed_rt',
-        'label_val_to_keep'
+        'label_val_to_keep',
+        "create_rows",
```

Comment:
The problem with CI is in adding the `create_rows` function to the cluster 

```
Error in `get(name, envir = envir)`: object 'create_rows' not found
```

I think we need to add `@export` to its definition. This can possibly apply also to some other added functions, needs to be double checked. 

> The CI output is not very explanatory, but it can be reproduced by running `devtools::check` locally.

Reply:
Yes, this was the issue. fixed in [c69fece](https://github.com/RECETOX/recetox-aplcms/pull/182/commits/c69fece7afe0e80dc80d9cc53fb81fe1a8a1dc53) thank you!